plasma_framework/contracts/src/vaults/EthVault.sol

```diff
@@ -10,6 +10,13 @@ contract EthVault is Vault {
         uint256 amount
     );
 
+    event DepositCreated(
+        address indexed depositor,
+        uint256 indexed blknum,
+        address indexed token,
```

Comment:
hmmm, do we really need this? I think the client would be forced to listen event from different contracts anyway, keeping the interface for both vault might not bring much value?

Reply:
I did it to be consistent with previous version of the contracts.